index.html

```diff
@@ -8,12 +8,121 @@
+                  </li>
+                  <li class="navigation-item">
+                    <a class="navigation-link" href="#">Blog</a>
+                  </li>
+                  <li class="navigation-item">
+                      <a class="navigation-link" href="#">Help</a>
+                    </li>
+                    <li class="navigation-item">
+                      <a class="navigation-link" href="#">Login</a>
+                    </li>
+                    </ul>
+              </nav>
+    </div>
+
+    <div class="mainImage">
+            <div class="centred">
+                <p class="para1">Introducing Karma</p>
+            </div>
+
+            <p class="para2">Bring WiFi with you, everywhere you go.</p>
+
+       
+            <div class="buttonContainer">
+                  <button class="LearnMoreButton">Learn More</button>
+            </div>
+    </div>
+
+    <section class="LowerHalf">
+        <p class="every1">Everyone needs a little Karma.</p>
+
+        <div class="iconFlexBox">
+              <div>
+                <img src="img\icon-devices.svg" alt="icons of computer, tablet and mobile" height=160px; class="iconPadding">
+                <p class="description">Internet for all devices</p> 
+              </div>
+
+              <div>
+                <img src="img\icon-coffee.svg" alt="icon of a cup of coffee" height=160px; class="iconPadding">
+                <p class="description">Boost your productivity</p> 
+              </div>
+
+              <div>
+                <img src="img\icon-refill.svg" alt="icon of a petrol pump" height=160px; class="iconPadding2">
+                <p class="description">Pay as You Go</p> 
+              </div>
+      </div>
+    </section>
+
+    <footer class="theFooter">
+
+      <div class="footer-content">
+        <hr class="drawaline">
+        <p class="joinUs">Join us on</p>
+
+          <div class="placeInTheCentre">
+               <div class="socialMediaFlexBox">
+                      <a href="#">
+                            <img src="img\twitter-icon.svg" alt="Twitter link"; height=20px; width=20px; class="socialMedia">
+                      
+                      </a>
+
+                      <a href="#">
+                            <img src="img\facebook-icon.svg" alt="Facebook link"; height=20px; width=20px; class="socialMedia">                        
+                      </a>
+                    
+                      <!-- Replace these 5 lines as shown below if ORANGE INSTAGRAM icon is required -->
+                      <a href="#">
+                          <div class="grayBorder">
+                            <img src="img\instagram-icon.svg" alt="Instagram link"; height=20px; width=20px; class="blueInstagram">   
+                          </div>                     
+                        </a>
+                     
+              </div>
+          </div>
+    
+         <div class="copyright">&copy; Karma Mobility, Inc.</div>
+    </div>
```

Comment:
This indentation doesn't look right. Each level of tag should result in an additional indent like so: 
```
<html>
  <body>
    <footer>
    </footer>
  </body>
</html>
```